backend/models/Comment.js

```diff
@@ -1,12 +1,13 @@
+// models/Comment.js
 import mongoose from 'mongoose';
 
 const commentSchema = new mongoose.Schema({
-  recipe: { type: mongoose.Schema.Types.ObjectId, ref: 'Recipe', required: true }, // Link to the related recipe
-  username: { type: String, required: true },     // User who made the comment
+  recipe: { type: mongoose.Schema.Types.ObjectId, ref: 'Recipe', required: true },
+  user: { type: mongoose.Schema.Types.ObjectId, ref: 'User', required: true }, // Link to the user who made the comment
+  username: { type: String, required: true }, 
   content: { type: String, required: true },
```

Comment:
Make sure this model change does not break any other API

Reply:
@AlfiyaSiddique this was wrongly commited, fixed them to match the upstream